Cargo.toml

```diff
@@ -22,6 +22,10 @@ structopt = "0.2"
 [profile.release]
 debug = true
 
+[lib]
+name = "inferno_collapse_perf"
```

Comment:
Given that we'll also expand this repository to have other programs, I wonder if I'd rather use nested modules here:
```rust
inferno::collapse::perf
```

Reply:
Agreed.